packages/ndla-ui/src/RelatedArticleList/RelatedArticleList.jsx

```diff
@@ -10,7 +10,7 @@ const classes = new BEMHelper({
   prefix: 'c-',
 });
 
-export const RelatedArticle = ({ title, introduction, icon, modifier, to, linkInfo }) => {
+export const RelatedArticle = ({ title, introduction, icon, modifier, to, linkInfo, target }) => {
```

Comment:
Kanskje defaulte til tom streng. Då treng du vel heller ikkje defaultProps?
```suggestion
export const RelatedArticle = ({ title, introduction, icon, modifier, to, linkInfo, target = '' }) => {
```